testkit/src/checkpoint_db.rs

```diff
@@ -308,11 +304,9 @@ mod tests {
             let inner = db.inner.read().unwrap();
             let stack = &inner.backup_stack;
             assert_eq!(stack.len(), 2);
-            let backup = stack.last().expect("There are not backups in the stack");
+            let backup = &stack[1];
             assert_eq!(backup.len(), 1);
```

Comment:
Tiny nitpick: you can write `assert_eq!(stack[1].len(), 1)` directly here and in similar places; imo, it doesn't hurt code readability.